indexes/badger/index.go

```diff
@@ -94,7 +95,7 @@ func newIndex(db *badger.DB, tipe interface{}, keyPrefix []byte) indexes.SeqSett
 		db:     db,
 		tipe:   tipe,
 		obvs:   make(map[indexes.Addr]luigi.Observable),
-		curSeq: margaret.BaseSeq(-2),
+		curSeq: int64(-2),
```

Comment:
even this small diff shows how much cleaner, conceptually, this pr is. new people no longer need to spend time or working memory slots considering _"what is a `BaseSeq`?"_

:100: